docs/machine-learning/tutorials/text-classification-tf.md

```diff
@@ -0,0 +1,253 @@
+---
+title: 'Tutorial: Analyze sentiment of movie reviews using a pre-trained TensorFlow model'
+description: This tutorial shows you how to use a pre-trained TensorFlow model to classify sentiment in website comments. The binary sentiment classifier is a C# console application developed using Visual Studio.
+ms.date: 09/11/2019
+ms.topic: tutorial
+ms.custom: mvc
+ms.author: nakersha
+author: natke
+#Customer intent: As a developer, I want to use ML.NET to make inferences with a pre-trained TensorFlow model. 
+---
+# Tutorial: Analyze sentiment of movie reviews using a pre-trained TensorFlow model in ML.NET
+
+This tutorial shows you how to use a pre-trained TensorFlow model to classify sentiment in website comments. The binary sentiment classifier is a C# console application developed using Visual Studio.
+
+The TensorFlow model used in this tutorial was trained using movie reviews from the IMDB database. Once you have finished developing the application, you will be able to supply movie review text and the application will tell you whether the review has positive or negative sentiment.
+
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> * Load a pre-trained TensorFlow model
+> * Transform website comment text into features suitable for the model
+> * Use the model to make a prediction
+
+You can find the source code for this tutorial at the [dotnet/samples](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/TextClassificationTF) repository.
+
+## Prerequisites
+
+* [Visual Studio 2017 15.6 or later](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) with the ".NET Core cross-platform development" workload installed.
+
+* [The sentiment_analysis machine learning model Zip file](https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model)
+
+## Setup
+
+### Create the application
+
+1. Create a **.NET Core Console Application** called "TextClassificationTF".
+
+2. Create a directory named *Data* in your project to save your data set files.
+
+3. Install the **Microsoft.ML NuGet Package**:
+
+    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the package source, and then select the **Browse** tab. Search for **Microsoft.ML**, select the package you want, and then select the **Install** button. Proceed with the installation by agreeing to the license terms for the package you choose. Repeat these steps for **Microsoft.ML.TensorFlow**.
+
+    > [!NOTE]
```

Comment:
Suggestion: Move this section into the "Add the TensorFLow model to the project" section

Reply:
Do you mean, addition of the Microsoft.ML.TensorFlow nuget? i.e. keep the main here and add the TensorFlow one below, or do you mean move both below?

Reply:
Sorry should've done it one line below for it to be clearer. I meant the NOTE. Move it to the next section.

Reply:
Oh right. Good call. Moving ...